### ISSUE TITLE
1.5.1, nightly and changelog

### ISSUE THREAD

**Issue by stepancheg** [rust-lang/rustfmt#5409]
```
% rustup update
...
     stable-x86_64-apple-darwin unchanged - rustc 1.61.0 (fe5b13d68 2022-05-18)
  stable-x86_64-pc-windows-msvc unchanged - (rustc does not exist)
    nightly-x86_64-apple-darwin unchanged - rustc 1.63.0-nightly (fdca237d5 2022-06-24)
```

```
$ rustc +nightly --version
rustc 1.63.0-nightly (fdca237d5 2022-06-24)
```

```
$ rustfmt +nightly --version
rustfmt 1.5.0-nightly (fdca237d 2022-06-24)
```

The version shown is 1.5.0.

Changelog entry for version 1.5.1:

> [1.5.1] 2022-06-24
> rustup (nightly) - nightly-2022-06-24

So either rustfmt prints version incorrectly, or the version was not updated in nightly-2022-06-24, as announced in changelog.

**Comment by calebcartwright** [comment]
Thanks, but closing as unnecessary. The nightly release process is strictly dictated by what happens to exist in the rust-lang/rust repo at the point in time the nightly builds kick off. When we make changes in this repository respective to dates about upstream releases, they are best guesses, as we have no future insight into when PRs upstream, in a different repository may land.

There was a pretty massive backup in the bors queue that delayed a number of PRs getting landed in rust-lang/rust, including the subtree sync that got the changes incorporated. It has finally landed (a few hours ago) so the next nightly build will contain the updated version of rustfmt. Believe that should be 06-26, but could potentially be 06-27 too.

Long story short, don't view upstream release dates in our changelog as unequivocal truth, especially so close to the dates. They're estimates, which we try to circle back to correct, but not always (you'll notice plenty of entries that don't even have a date).
